content/en/logs/workspaces/export.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Export Log Workspaces Datasets
+private: true
+further_reading:
+- link: "/logs/workspaces/"
+  tag: "Documentation"
+  text: "Learn more about Log Workspaces"
+- link: "/dashboards/widgets/list/"
+  tag: "Documentation"
+  text: "List Widget"
+---
+
+## Overview
+
+Export your Workspaces cell as a dataset to dashboards and notebooks. The export feature allows you to create datasets and display them in dashboards and notebooks for analysis or reporting. 
+
+## Export a Workspaces cell as a dataset 
+
+From your workspace:
+
+1. Find the dataset you want to export.  
```

Comment:
1. Find the cell you want to export as a dataset.

Reply:
```suggestion
1. Find the cell you want to export as a dataset.  
```